rose/server/score.py

```diff
@@ -40,27 +40,26 @@ def process(players, track):
             if player.action != actions.JUMP:
                 track.clear(player.x, player.y)
                 player.y += 1
-                player.score += config.score_move_backward
+                player.score += config.score_move_backward * 2
```

Comment:
Why is it better to give more score?
Why not change the config so config.score_move_backward is larger?

I don't think this is a good change, it makes configuration harder since
we need to consider this magic hidden factor.

It is better to change the configuration instead of the code.

Reply:
It was intended to cancel out the score for moving forward, they are the same. Will fix to removing the forward score.